app/views/messages/_messages_for_library.html.erb

```diff
@@ -1,24 +1,48 @@
 <div class="messages">
-<% messages = @messages.select { |m| m.library == library_code && m.request_type == request_type } %>
-<% messages.each do |message| %>
+
+  <% messages = @messages.select { |m| m.library == library_code && m.request_type == request_type } %>
+
+  <% if library_code == 'SAL' %>
+    <% library_code = "SAL1&2" %>
+  <% elsif library_code == "SAL-NEWARK"%>
+    <% library_code = "SAL Newark" %>
+  <% end %>
+
+  <h2><%= request_type.titleize + " from " + library_code if messages.none? %></h2>
+  <%= link_to " Add message", new_message_path(library: library_code, request_type: request_type), class: 'btn btn-sm btn-default edit-message glyphicon glyphicon-pencil' if messages.none? %>
```

Comment:
It seems like this `link_to` is identical to the one below.  Maybe these should be in a helper so we don't violate DRY.